### PR TITLE
Add linting

### PR DESCRIPTION
**Pull request by dustincoates**

### CODE REVIEW

**Comment by dustincoates** [comment]
@kokliKo Still getting these errors on the geo search example:

```
/Users/dustin/Documents/code/examples/geo-search-demo/js/app.js
  179:5   warning  Unexpected console statement            no-console
  307:11  error    "infoWindow" is defined but never used  no-unused-vars
```

How do you want to fix these? Especially on the second, I'm not sure if that's necessary or not.

File: variants_colors/js/app.js [hunk 1]
```diff
@@ -79,36 +72,32 @@ $(document).ready(function() {
...
     $('#query-message').empty();
     setDistinct($(this));
     algoliaHelper.search();
   });
-  $(document).on('click', '.set-query', function(e) {
+  $(document).on('click', '.set-query', function (e) {
     e.preventDefault();
     $('#query-message').html($(this).data('message'));
-    setDistinct($(".set-distinct[data-value='"+$(this).data('distinct')+"']"));
+    setDistinct($(".set-distinct[data-value='" + $(this).data('distinct') + "']"));
```
Comment (vvo) [inline variants_colors/js/app.js]: weird I thought we were forcing single quotes

File: geo-search-demo/js/app.js [hunk 2]
```diff
@@ -278,25 +294,34 @@ $(document).ready(function() {
...
+
+    if (hit.name === hit.city) {
+      message = hit.name + ' - ' + hit.country;
+    } else {
+      message = hit.name + ' - ' + hit.city + ' - ' + hit.country;
+    }
+
+    var infowindow = new google.maps.InfoWindow({content: message});
+    marker.addListener('click', function () {
+      var infoWindow = infowindow.open(marker.get('map'), marker);
```
Comment (vvo) [inline geo-search-demo/js/app.js]: You can just do

```js
infowindow.open(marker.get('map'), marker);
```

I think, no need to store into a variable

File: variants_colors/js/app.js (hunk 1 above)
Comment (vvo) [inline variants_colors/js/app.js]: This should fail

Comment (vvo) [inline variants_colors/js/app.js]: OH Wait, I know why it works, because you have single quotes inside it, nevermind

**Comment by vvo** [comment]
> 179:5   warning  Unexpected console statement            no-console

Maybe just ignore the error for this one since it's an interesting to have it here, it shows the user he should care about errors.

File: geo-search-demo/js/app.js [hunk 3]
```diff
@@ -132,34 +150,32 @@ $(document).ready(function() {
...
   // ===============
-  algoliaHelper.on('result', function(content, state) {
+  algoliaHelper.on('result', function (content) {
     renderMap(content);
     renderHits(content);
   });
 
-  algoliaHelper.on('error', function(error) {
+  algoliaHelper.on('error', function (error) {
     console.log(error);
```
Comment (vvo) [inline geo-search-demo/js/app.js]: to ignore the console warning:

```js
function (error) {
  /*eslint-disable no-console */
 ... code
}
```

File: geo-search-demo/js/app.js (hunk 2 above)
Comment (dustincoates) [inline geo-search-demo/js/app.js]: Yeah, I was curious about that. It was a global var before, so I'm not sure if it's needed anywhere. @kokliKo thoughts?

Comment (vvo) [inline geo-search-demo/js/app.js]: could not find any reference (+ if it works, it works :D)

**Pull request merged**